person.rb

```diff
@@ -19,3 +30,28 @@ def of_age?
     @age >= 18
   end
 end
+
+class Decorator < Nameable
+  def initialize(nameable)
+    super()
+    @nameable = nameable
+  end
+
+  def correct_name
+    @nameable.correct_name
+  end
+end
+
+class CapitalizeDecorator < Decorator
+  def correct_name
+    super.capitalize
+  end
+end
+
+class TrimmerDecorator < Decorator
+  MAX_LENGTH = 10
+
+  def correct_name
+    super[0, MAX_LENGTH]
+  end
```

Comment:
- [ ] [OPTIONAL] Kindly move the `Nameable`, `Decorator`, `CapitalizeDecorator`, and `TrimmerDecorator` classes to separate files in order to make your code clean and professional.